src/bidiMapper/domains/context/context.ts

```diff
@@ -82,6 +96,36 @@ export abstract class Context {
     this.#parentId = parent;
     this.#sessionId = sessionId;
     this.cdpClient = cdpClient;
+
+    this.setCDPListeners();
```

Comment:
In order to follow http://go/jsstyle (open source: https://google.github.io/styleguide/jsguide.html), this should be `setCdpListeners` c.f. https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers

Reply:
done